### ISSUE TITLE
Parsing error upon "Fetch details for attempt ..."

### ISSUE THREAD

**Issue by christofferqa** [Mortal/bbfetch#5]
The command ´./grading -d´ fails with a Parsing error:

```
christofferqa:handins cqa$ ./grading -d
[2016-12-14 08:58:58,349 INFO] Refresh gradebook
[2016-12-14 08:58:59,751 INFO] Fetch details for attempt Group Attempt Gruppe 1 - 02 11-12-16
[2016-12-14 08:59:00,604 ERROR] Parsing error
No currentAttempt_submissionList
ParserError logged to 2016-12-14_0859_parseerror.txt
```

The relevant group has made an attempt with a comment, but has forgotten to upload the actual handin. Grading the attempt in Blackboard, does not seem to remove the problem.

**Comment by Mortal** [comment]
This sounds like a duplicate of #4 which was fixed with 74e4c18064ac231eb66c8a6a6da91eb264d8803b. Do you have that commit in your local repo?

After grading the attempt manually on the Blackboard website, try running ./grading -a before running ./grading -d.

**Comment by christofferqa** [comment]
I was missing the commit, sorry for the confusion!
